datadog/resource_datadog_dashboard.go

```diff
@@ -3297,6 +3297,10 @@ func getQueryTableDefinitionSchema() map[string]*schema.Schema {
 				Schema: getWidgetTimeSchema(),
 			},
 		},
+		"has_search_bar": {
+			Type:     schema.TypeString,
```

Comment:
Should this be a bool?

Reply:
Ahh ok thanks. You could do something like set a ValidateFunc with stringInSlice - https://github.com/hashicorp/terraform/blob/master/helper/validation/validation.go#L32 but I think its OK to just document the allowed fields too.